server/datastore/mysql/apple_mdm.go

```diff
@@ -3081,6 +3081,49 @@ WHERE h.uuid = ?
 	return nil
 }
 
+func (ds *Datastore) MDMAppleDDMSynchronizationTokens(ctx context.Context, teamID uint) (*fleet.MDMAppleDDMSyncTokens, error) {
+	const stmt = `
+SELECT
+	md5_checksum,
+	latest_created_timestamp
+FROM
+	team_declaration_checksum_view
+WHERE
+	team_id = ?
+`
+
+	var res fleet.MDMAppleDDMSyncTokens
+	if err := sqlx.GetContext(ctx, ds.reader(ctx), &res, stmt, teamID); err != nil {
+		return nil, ctxerr.Wrap(ctx, err, "get DDM checksum by team id")
+	}
+
+	return &res, nil
+}
+
+func (ds *Datastore) MDMAppleDDMDeclarationItems(ctx context.Context, teamID uint) ([]fleet.MDMAppleDDMDeclarationItemDB, error) {
+	// TODO: Confirm whether we can use JSON functions in the query (if 5.7 officially unsupported
+	// by Fleet)
+	const stmt = `
+SELECT
+	mad.md5_checksum as server_token,
+	identifier,
+	declaration_type,
```

Comment:
per discussion in Slack, we realized that his needs to be modified to be the actual `Type` that's inside the JSON declaration.

Reply:
I was wrong, and this is done by identifier. Marking as solved